pontoon/base/static/js/fluent_interface.js

```diff
@@ -258,7 +258,13 @@ var Pontoon = (function (my) {
 
         elements.forEach(function (item) {
           if (item.type === 'TextElement') {
-            translatedValue += item.value;
+            // TODO: We shouldn't rely on markPlaceables in determining when to not render HTML.
+            if (markPlaceables) {
+              translatedValue += Pontoon.doNotRender(item.value);
+            }
+            else {
```

Comment:
`} else {`

Reply:
I'm keeping the code style consistent with how @adngdb reformatted the file. I like his code style a lot, but I agree this particular case looks odd.

Reply:
Ah, this is another case where it's all about personal taste... I personally dislike `} else {`, I find it harder to identify blocks and logic with it. In general I'm a big fan of vertical space, and every attempt at removing some makes me sad. :)